cmd/block-generator/generator/generate.go

```diff
@@ -637,3 +650,82 @@ func (g *generator) generateAssetTxn(round uint64, intra uint64) (transactions.S
 
 	return signTxn(txn), transactions.ApplyData{}, nil
 }
+
+func (g *generator) WriteAccount(output io.Writer, accountString string) error {
+	addr, err := basics.UnmarshalChecksumAddress(accountString)
```

Comment:
I started with that, since it makes sense to validate the data early. Ended up with this because I'm using the encoded and decoded version so I avoid re-encoding.